.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Correct if wrong, but JDK_1_8 doesn't support Lambda Expressions. The ViewModel utilizes the Lambda expressions capability to push Data transaction work off of the main thread. If the JDK downgrade doesn't effect functionality, this change is conditionally approved.

Reply:
Strange, I never intentionally changed the language level, good catch. I will fix and recommit